### PR TITLE
Audit name

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

- Has the front end post an `auditName` field to the `/election/new` endpoint

**Testing**

- Tests that it passes as expected for both authenticated and unauthenticated users, and is required
- Updates snapshots

**Progress**

- Do we need to verify the uniqueness of the audit name at this time?
- The back end needs to use the value as passed back when creating an audit, still

image

### CODE REVIEW

**Comment by MorganLove** [comment]
I think this PR should cover everything that closes #349 once it's done.

File: arlo-client/src/components/CreateAudit.test.tsx [hunk 1]
```diff
@@ -86,15 +104,20 @@ describe('CreateAudit', () => {
...
 
     await wait(() => {
       expect(apiMock).toBeCalledTimes(2)
       expect(apiMock).toHaveBeenNthCalledWith(2, '/election/new', {
         method: 'POST',
-        // eslint-disable-next-line @typescript-eslint/camelcase
-        body: JSON.stringify({ organization_id: 'org-id' }),
+        body: JSON.stringify({
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          organization_id: 'org-id',
```
Comment (jonahkagan) [inline arlo-client/src/components/CreateAudit.test.tsx]: Oops, should we make a follow up task to camelCase this on the backend?

Comment (MorganLove) [reply arlo-client/src/components/CreateAudit.test.tsx]: @jonahkagan I think @eventualbuddha already created a PR to do that, just need to update this to match when it gets rebased.

Comment (eventualbuddha) [reply arlo-client/src/components/CreateAudit.test.tsx]: Yep, this was merged in #353.

Comment (jonahkagan) [reply arlo-client/src/components/CreateAudit.test.tsx]: giphy

File: arlo-client/src/components/CreateAudit.tsx [hunk 2]
```diff
@@ -60,18 +71,37 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
   return (
     <Wrapper>
       <img height="50px" src="/arlo.png" alt="Arlo, by VotingWorks" />
-      <Button
-        verticalSpaced
```
Comment (eventualbuddha) [inline arlo-client/src/components/CreateAudit.tsx]: This line seems to have been lost when I rebased. Do we want it, or is it right that it's gone? @MorganLove

Comment (MorganLove) [reply arlo-client/src/components/CreateAudit.tsx]: @jonahkagan That works with it being gone. I just compared with and without and it looks better without. Thanks for checking! 👍

**Comment by MorganLove** [comment]
@eventualbuddha Any idea why this is failing backend tests now?

**Pull request merged**